pkg/microservice/aslan/core/environment/service/environment.go

```diff
@@ -445,6 +448,19 @@ func UpdateProduct(existedProd, updateProd *commonmodels.Product, renderSet *com
 		return e.ErrUpdateEnv.AddErr(err)
 	}
 
+	cls, err := kubeclient.GetKubeClientSet(config.HubServerAddress(), existedProd.ClusterID)
+	if err != nil {
+		log.Errorf("[%s][%s] error: %v", envName, namespace, err)
+		err = e.ErrUpdateEnv.AddDesc(err.Error())
+		return
```

Comment:
the err variable here is re-declared in line 451, which means the return err value is nil